src/test/server/java/ch/zhaw/iwi/cis/kelvin/test/server/TestConfig.java

```diff
@@ -8,6 +8,11 @@
 @ApplicationConfig
 public class TestConfig extends KelvinConfig
 {
+	public String getPluginDir()
```

Comment:
Mark this with @Override annotation; configure Eclipse to generate warning when missing.